tools/fiptool/fiptool.h

```diff
@@ -10,8 +10,8 @@
 #include <stddef.h>
 #include <stdint.h>
 
-#include "firmware_image_package.h"
-#include "uuid.h"
+#include <firmware_image_package.h>
```

Comment:
I am opposed to the re-sort for the same reason above.
The current include order already makes sense.

I update the copyright date only for big changes (because it is kind of a noise), but I do not have a strong opinion.
I can follow the maintainers here.